### ISSUE TITLE
Aborterror: Registration failed - push service error

### ISSUE THREAD

**Issue by thunderbug1** [gauntface/simple-push-demo#377]
Trying the demo page on chrome desktop works, but on chrome mobile on android I always get:

Aborterror: Registration failed - push service error

![Screenshot_20231101-165722](https://github.com/gauntface/simple-push-demo/assets/1395321/6b42c61c-a366-4208-8627-e4a213008b92)

**Comment by gauntface** [comment]
If possible, could you share the following:

1. Version of Chrome
1. Version of Android
1. What country you are in
1. What device you are using

**Comment by thunderbug1** [comment]
Sure, I am currently in Spain. This ouput of chrome://version should contain all the info:



Google Chrome | 118.0.5993.111 (Official Build) (64-bit)
-- | --
Revision | 47c097e133ae535cd69c530a3c9a48e8fff3c825-refs/branch-heads/5993@{#1392}
OS | Android 14; Pixel 6 Build/UP1A.231005.007; 34; REL
APK versionCode | 599311133
APK targetSdkVersion | 34
isAtLeastU/targetsAtLeastU | true/true
Google Play services | SDK=233700000; Installed=234212044; Access=1p
JavaScript | V8 11.8.172.16
User Agent | Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Mobile Safari/537.36

**Comment by thunderbug1** [comment]
I made a minimum flutter example for firebase messaging for the same error, not sure if it is related:
https://github.com/firebase/flutterfire/issues/11818

**Comment by gauntface** [comment]
I just found that the same happens in Brave browser on Mac OS X.

**Comment by thunderbug1** [comment]
Yes for Brave I found an old Issue where they said that Brave disables Google Push Servers by default for privacy reasons. https://github.com/firebase/firebase-js-sdk/issues/3195

**Comment by gauntface** [comment]
For anyone looking in the future - you can enable the use of Google push services in brave.
